dli_app/templates/auth/login.html

```diff
@@ -28,5 +28,6 @@
         </div>
       </div>
     </form>
+    <a href="http://68.234.146.84:8222/auth/resetpass/">I forgot my password</a>
```

Comment:
Don't hardcode this. This is where jinja2 shines.

```python
<a href="{{ url_for('auth.resetpass') }}">
```

That way if we ever rename the resetpass method, our templates will automatically update.